test/integration/targets/win_dns_record/tasks/tests.yml

```diff
@@ -1,3 +1,13 @@
+- name: ensure DNS services are installed
+  win_feature:
+    name: DNS
+    state: present
+  register: dns_install
+
+- name: reboot server if needed
+  win_reboot:
+  when: dns_install.reboot_required
```

Comment:
@johnboy2. Maybe better Asking for changes here? like:

https://github.com/ansible/ansible/blob/0e980596376be7b7ef843d20b52228687e94717d/test/integration/targets/win_dns_record/tasks/main.yml#L16-L18

Reply:
Ooops, you just changed it. :innocent: Though this was other file.

Reply:
:-)